webui/react/src/components/Table.tsx

```diff
@@ -141,10 +154,24 @@ export const defaultRowClassName = (clickable = true): string=> {
   return clickable ? 'clickable' : '';
 };
 
-export const getPaginationConfig = (count: number): TablePagination => {
+export const getPaginationConfig = (count: number): Partial<TablePaginationConfig> => {
   return {
     defaultPageSize: MINIMUM_PAGE_SIZE,
     hideOnSinglePage: count < MINIMUM_PAGE_SIZE,
     showSizeChanger: true,
   };
 };
+
+export const getFullPaginationConfig = (
+  pagination: TablePagination,
+  total: number,
+): TablePaginationConfig => {
+  return {
+    current: Math.floor(pagination.offset / pagination.limit) + 1,
+    defaultPageSize: MINIMUM_PAGE_SIZE,
+    hideOnSinglePage: total < MINIMUM_PAGE_SIZE,
```

Comment:
question: If the page size changes, does this value need to change too?

Reply:
the `defaultPageSize` remains the same. There is a `pageSize` attribute that changes along with the page size picker. The `hideOnSinglePage` basically determines whether to show or hide the entire pagination component. The bar is low and we just check to make sure we have at least `MINIMUM_PAGE_SIZE` (10) items. If this is dynamic, it can cause pagination to be hidden when going from page size of 20 to 10.